VnV/Vehicles/Tiltwing_EVTOL.py

```diff
@@ -318,7 +318,8 @@ def vehicle_setup(new_regression=True):
         design_electric_rotor(regression_prop_rotor_propulsor, iterations=2)
         loaded_propulsor = load_propulsor(os.path.join(test_dir, 'vahana_tilt_rotor_propulsor.res'))  
         for key,item in prop_rotor_propulsor.rotor.items():
-            prop_rotor_propulsor.rotor[key] = loaded_propulsor.rotor[key] 
+            if key != "volume_properties":
+                prop_rotor_propulsor.rotor[key] = loaded_propulsor.rotor[key] 
```

Comment:
This is buggy and the start of something we don't want to go down. I suggest you remove these lines or find a clearer fix